Core unit test failure (core_cache\administration_helper_test::test_get_summaries)

Pull request #1451 caused a regression resulting in a core unit test failure.

```
php vendor/bin/phpunit --testsuite core_cache_testsuite --filter test_get_summaries
Moodle 4.5.6+ (Build: 20250912), a403ab4b0bab410ad592d3cf6b61cb6cc98239ee
Php: 8.1.33, pgsql: 17.6 (Debian 17.6-1.pgdg13+1), OS: Linux 6.8.0-79-generic x86_64
PHPUnit 9.6.18 by Sebastian Bergmann and contributors.

E                                                                   1 / 1 (100%)

Time: 00:00.076, Memory: 60.50 MB

There was 1 error:

1) core_cache\administration_helper_test::test_get_summaries
Unexpected debugging() call detected.
Debugging: String does not exist. Please check your string definition for cachedef_parsercache/qtype_stack
* line 164 of /lib/classes/lang_string.php: call to debugging()
* line 612 of /cache/classes/definition.php: call to core\lang_string->__construct()
* line 191 of /cache/classes/administration_helper.php: call to core_cache\definition->get_name()
* line 98 of /cache/tests/administration_helper_test.php: call to core_cache\administration_helper::get_definition_summaries()
* line 1617 of /vendor/phpunit/phpunit/src/Framework/TestCase.php: call to core_cache\administration_helper_test->test_get_summaries()
* line 1223 of /vendor/phpunit/phpunit/src/Framework/TestCase.php: call to PHPUnit\Framework\TestCase->runTest()
* line 76 of /lib/phpunit/classes/advanced_testcase.php: call to PHPUnit\Framework\TestCase->runBare()
* line 729 of /vendor/phpunit/phpunit/src/Framework/TestResult.php: call to advanced_testcase->runBare()
* line 973 of /vendor/phpunit/phpunit/src/Framework/TestCase.php: call to PHPUnit\Framework\TestResult->run()
* line 685 of /vendor/phpunit/phpunit/src/Framework/TestSuite.php: call to PHPUnit\Framework\TestCase->run()
* line 685 of /vendor/phpunit/phpunit/src/Framework/TestSuite.php: call to PHPUnit\Framework\TestSuite->run()
* line 685 of /vendor/phpunit/phpunit/src/Framework/TestSuite.php: call to PHPUnit\Framework\TestSuite->run()
* line 651 of /vendor/phpunit/phpunit/src/TextUI/TestRunner.php: call to PHPUnit\Framework\TestSuite->run()
* line 146 of /vendor/phpunit/phpunit/src/TextUI/Command.php: call to PHPUnit\TextUI\TestRunner->run()
* line 99 of /vendor/phpunit/phpunit/src/TextUI/Command.php: call to PHPUnit\TextUI\Command->run()
* line 107 of /vendor/phpunit/phpunit/phpunit: call to PHPUnit\TextUI\Command::main()
* line 121 of /vendor/bin/phpunit: call to include()
Debugging: String does not exist. Please check your string definition for cachedef_librarycache/qtype_stack
* line 164 of /lib/classes/lang_string.php: call to debugging()
* line 612 of /cache/classes/definition.php: call to core\lang_string->__construct()
* line 191 of /cache/classes/administration_helper.php: call to core_cache\definition->get_name()
* line 98 of /cache/tests/administration_helper_test.php: call to core_cache\administration_helper::get_definition_summaries()
* line 1617 of /vendor/phpunit/phpunit/src/Framework/TestCase.php: call to core_cache\administration_helper_test->test_get_summaries()
* line 1223 of /vendor/phpunit/phpunit/src/Framework/TestCase.php: call to PHPUnit\Framework\TestCase->runTest()
* line 76 of /lib/phpunit/classes/advanced_testcase.php: call to PHPUnit\Framework\TestCase->runBare()
* line 729 of /vendor/phpunit/phpunit/src/Framework/TestResult.php: call to advanced_testcase->runBare()
* line 973 of /vendor/phpunit/phpunit/src/Framework/TestCase.php: call to PHPUnit\Framework\TestResult->run()
* line 685 of /vendor/phpunit/phpunit/src/Framework/TestSuite.php: call to PHPUnit\Framework\TestCase->run()
* line 685 of /vendor/phpunit/phpunit/src/Framework/TestSuite.php: call to PHPUnit\Framework\TestSuite->run()
* line 685 of /vendor/phpunit/phpunit/src/Framework/TestSuite.php: call to PHPUnit\Framework\TestSuite->run()
* line 651 of /vendor/phpunit/phpunit/src/TextUI/TestRunner.php: call to PHPUnit\Framework\TestSuite->run()
* line 146 of /vendor/phpunit/phpunit/src/TextUI/Command.php: call to PHPUnit\TextUI\TestRunner->run()
* line 99 of /vendor/phpunit/phpunit/src/TextUI/Command.php: call to PHPUnit\TextUI\Command->run()
* line 107 of /vendor/phpunit/phpunit/phpunit: call to PHPUnit\TextUI\Command::main()
* line 121 of /vendor/bin/phpunit: call to include()


/var/www/site/lib/phpunit/classes/advanced_testcase.php:101

ERRORS!
Tests: 1, Assertions: 53, Errors: 1.

```

--- a/lang/en/qtype_stack.php
+++ b/lang/en/qtype_stack.php
@@ -30,6 +30,9 @@ $string['pluginnameediting'] = 'Editing a STACK question';
 $string['pluginnamesummary'] = 'STACK provides mathematical questions for the Moodle quiz.  These use a computer algebra system to establish the mathematical properties of the student\'s responses.';
 
 $string['privacy:metadata']  = 'The STACK question type plugin does not store any personal data.';
+$string['cachedef_parsercache'] = 'STACK parsed Maxima expressions';
+$string['cachedef_librarycache'] = 'STACK question library renders and file structure';
+
 $string['mbstringrequired'] = 'Installing the MBSTRING library is required for STACK.';
 $string['yamlrecommended']  = 'Installing the YAML library is recommended for STACK.';
 

